docs/more-pushmsg.md

```diff
@@ -115,3 +115,101 @@ axios({
     }
   });
 ```
+
+### Java
+
+With maven dependency:
+```xml
+<dependency>
+    <groupId>com.fasterxml.jackson.core</groupId>
+    <artifactId>jackson-databind</artifactId>
+    <version>2.9.8</version>
+</dependency>
+```
+And code:
+```java
+package com.gotify.client;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class GotifyClient {
+
+    // example main method
+    public static void main(String[] args) throws IOException {
+        GotifyClient client = new GotifyClient("http://localhost:8008/message?token=<apptoken>");
+        client.sendMessage("Hi there", "Hello from Java", 5);
```

Comment:
We can use the message directly as input parameter. This would simplify the code IMO.